bigtable/src/test/java/com/m/examples/bigtable/QuickstartTest.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.m.examples.bigtable;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+
+import com.google.cloud.bigtable.admin.v2.BigtableTableAdminClient;
+import com.google.cloud.bigtable.admin.v2.BigtableTableAdminSettings;
+import com.google.cloud.bigtable.data.v2.BigtableDataClient;
+import com.google.cloud.bigtable.data.v2.BigtableDataSettings;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+import org.hamcrest.CoreMatchers;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.AssumptionViolatedException;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+/** Integration tests for {@link Quickstart} */
+public class QuickstartTest {
+  private static void requireSysProp(String varName) {
+    assertNotNull(
+        System.getenv(varName),
+        "System property '%s' is required to perform these tests.".format(varName));
+  }
+
+  @BeforeClass
+  public static void checkRequirements() {
+    requireSysProp(PROJECT_PROPERTY_NAME);
+    requireSysProp(INSTANCE_PROPERTY_NAME);
+  }
+
+  private static final String PROJECT_PROPERTY_NAME = "bigtable.project";
+  private static final String INSTANCE_PROPERTY_NAME = "bigtable.instance";
+  private static final String TABLE_ID = "quickstart-table";
+  private static BigtableDataClient dataClient;
+  private static BigtableTableAdminClient adminClient;
+  private static String projectId;
+  private static String instanceId;
+  private ByteArrayOutputStream bout;
+
+  @BeforeClass
+  public static void beforeClass() throws IOException {
+    projectId = System.getProperty(PROJECT_PROPERTY_NAME);
```

Comment:
Preferably these should be environment variables instead of system properties.

Reply:
Should I define the env variables here? https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/.kokoro/tests/run_tests.sh

There don't seem to be any product/test specific env variables, so I wasn't sure

Reply:
Sorry for the late response -- right now we have files that are sourced to add secrets too as done [here](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/21c141accd4870b27fd890aa3a9ea1b5cf84d712/.kokoro/tests/run_tests.sh#L58). If you send me an email with the credentials you would like to be added, I can add a file to the correct bucket.

Reply:
Fixed for this test and the other